### DISCUSSION TITLE
Why does no one use these

### DISCUSSION THREAD

**Discussion by Cuteistfox** [zws-im/zws#747] [Q&A] [answered]
Also are urls that are on backlisted domains before they were backlisted deleted to save space on the db

**Comment by jonahsnider** [comment, marked as answer]
> Why does no one use these

Not much to talk about.

> Also are urls that are on backlisted domains before they were backlisted deleted to save space on the db

Blocked URLs can't be shortened, but if an already shortened URL becomes blocked, it won't be deleted. This helps avoid accidentally deleting URLs that were erroneously marked blocked. It also means I can use the data for analytics like "when were the most blocked URLs shortened", which wouldn't be possible if blocked URLs were purged from the DB.
